components/graphs/ThresholdGraph.tsx

```diff
@@ -25,19 +25,23 @@ import {
 } from 'lib/utils';
 
 import { useFutureGraphData } from 'lib/hooks';
+import useTime from '~/lib/hooks/useTime';
 
 const ThresholdGraph = () => {
   const { data, isLoading } = useFutureGraphData();
-  const [currentTime, setCurrentTime] = useState(formatter.format(Date.now()));
+  // const [currentTime, setCurrentTime] = useState(formatter.format(Date.now()));
```

Comment:
All these commented lines can be removed before merge (this component becomes significantly simpler)